lib/node_modules/@stdlib/blas/base/dgemv/lib/ndarray.js

```diff
@@ -73,10 +73,10 @@ function dgemv( trans, M, N, alpha, A, strideA1, strideA2, offsetA, x, strideX,
 		throw new RangeError( format( 'invalid argument. Third argument must be a nonnegative integer. Value: `%d`.', N ) );
 	}
 	if ( strideX === 0 ) {
-		throw new RangeError( format( 'invalid argument. Tenth argument must be non-zero.' ) );
+		throw new RangeError( format( 'invalid argument. Tenth argument must be non-zero.', strideX ) );
 	}
 	if ( strideY === 0 ) {
-		throw new RangeError( format( 'invalid argument. Fourteenth argument must be non-zero.' ) );
+		throw new RangeError( format( 'invalid argument. Fourteenth argument must be non-zero.', strideY ) );
```

Comment:
```suggestion
		throw new RangeError( format( 'invalid argument. Fourteenth argument must be non-zero. Value: `%d`.', strideY ) );
```

You need to include the specifiers to interpolate.